content/blog/2022/03/Optimizing-Camel-K-Integration-Build-Time/index.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Optimizing Camel-K Integration Build Time"
+date: 2022-06-24
+draft: true
+authors: [SubhasmitaSw][astefanutti]
+categories: ["Camel-K"]
+preview: "Using performance benchmarks to demonstrate the optimised camel-k integration build time."
```

Comment:
I'd suggest "Scaling the Camel K operator vertically to improve the integration build time".